_posts/2020-07-12-NUnit3TestAdapter-4.0.0-alpha.1.md

```diff
@@ -0,0 +1,24 @@
+# NUnit3TestAdapter version 4.0.0-alpha.1 is released.
+
+This is an early pre-release version.
+
+The code has been rewritten/refactored in order to get some of the more complex issues fixed. It does pass all the automatic tests we have, but there are still more tests we would like to take it through, before we release a beta.
+
+We would really appreciate it if you give this alpha a spin. and report whatever you find back to us.
+
+The major fixes now are **Explicit** runs are fully back, both in Visual Studio and on command line with dotnet test and vstest.console.
```

Comment:
This sentence sounds odd to me (and I'm actually unsure exactly what we want to communicate)
```suggestion
The major fix in this release is to bring back the **Explicit** test runs, both in Visual Studio and on the command line with dotnet test and vstest.console. In addition to this, the implementation handles even more cases than before.
```